CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How to Contribute to Codecademy Community Bot
+
+All input is welcomed! Below are some of the basic steps to contribute to this project. Please read all of it to make sure you stay consistent with everybody else.
+
+Please review the _Issues_ tab and self-assign or ask to be assigned to one if you want to try to help in those. This keeps the efforts tidy and we know who is working on what.
+
+## Initial Setup
+
+If you don't have it already, install/setup Git on your machine.
+
+### Clone This Repo
```

Comment:
I'm wondering if we should change this to Clone or Fork This Repo, since if a non-org member clones and tries to push their changes it won't work.  We could add in a sentence that says if you're a member of our organization, clone it, but if you're new to the project, fork it.  If you become a regular contributor, we will add you to the organization so you can work off a direct clone instead.